google/cloud/storage/internal/retry_object_read_source.cc

```diff
@@ -52,7 +53,10 @@ RetryObjectReadSource::RetryObjectReadSource(
       offset_direction_(request_.HasOption<ReadLast>() ? kFromEnd
                                                        : kFromBeginning),
       current_offset_(InitialOffset(offset_direction_, request_)),
-      backoff_(std::move(backoff)) {}
+      backoff_(std::move(backoff)) {
+  backoff_ = google::cloud::internal::MakeTracedSleeper(
```

Comment:
optional: do this in the member initializer?

Reply:
Done.